Wikipedia/Code/OnThisDayExploreCollectionViewCell.swift

```diff
@@ -2,25 +2,38 @@ import UIKit
 
 @objc(WMFOnThisDayExploreCollectionViewCell)
 class OnThisDayExploreCollectionViewCell: OnThisDayCollectionViewCell {
-
-    private lazy var whiteTopAndBottomGradientLayer: CAGradientLayer = {
-        let layer = CAGradientLayer()
-        let white = UIColor.white.cgColor
-        let clear = UIColor(white: 1, alpha: 0).cgColor // https://stackoverflow.com/a/24895385
-        layer.colors = [white, clear, clear, white, white]
-        layer.locations = [0.0, 0.07, 0.77, 0.97, 1.0]
-        layer.startPoint = CGPoint(x: 0.5, y: 0.0)
-        layer.endPoint = CGPoint(x: 0.5, y: 1.0)
-        return layer
-    }()
-
+    fileprivate var topGradientView: WMFGradientView = WMFGradientView()
```

Comment:
@montehurd having two separate views reduces the amount of blending necessary to draw the cell. Compare simulator screenshots with "Color blended layers" enabled (other changes eliminated the blending for the labels, timeline view, tab bar, etc) :

Before:
![simulator screen shot jul 7 2017 7 05 49 am](https://user-images.githubusercontent.com/741327/27955770-615e5cb8-62e4-11e7-8987-25c807ee8705.png)

After:
![simulator screen shot jul 7 2017 7 12 57 am](https://user-images.githubusercontent.com/741327/27955737-3c8efd02-62e4-11e7-8fac-d1ae47530f1d.png)

Modern devices don't seem to gain any real improvement from this, but it should help scrolling performance on older devices.